llvm/unittests/ADT/APIntTest.cpp

```diff
@@ -3249,11 +3249,22 @@ TEST(APIntTest, SolveQuadraticEquationWrap) {
 }
 
 TEST(APIntTest, MultiplicativeInverseExaustive) {
```

Comment:
This is nowhere near "exhaustive" since it only tests one (power of two) modulus per BitWidth.